lib/views/category/edit_category_view.dart

```diff
@@ -7,25 +7,21 @@ import 'package:flipper/routes/router.gr.dart';
 import 'package:customappbar/customappbar.dart';
 import 'package:flipper/generated/l10n.dart';
 
-
 import 'package:flipper/views/category/category_viewmodel.dart';
 
 enum CategoriesEnum { beverage, drinks, ikawa }
 
 class EditCategoryScreen extends StatefulWidget {
```

Comment:
Let's avoid stateful widget and only rebuild UI only when it is needed @tuganimana